app/Console/Commands/FixIndicative.php

```diff
@@ -0,0 +1,168 @@
+                ['required']
+            );
+
+            $aidstreamOrganizationIds = explode(',', $aidstreamOrganizationIdString);
+
+            foreach ($aidstreamOrganizationIds as $key => $aidstreamOrganizationId) {
+                $aidstreamOrganizationIds[$key] = (int) $aidstreamOrganizationId;
+            }
+
+            $this->databaseManager->beginTransaction();
+
+            $aidstreamOrganizationIdentifierArray = $this->getAidstreamOrganizationIdentifier($aidstreamOrganizationIds);
+            $aidstreamOrganizationIdentifierArray = array_map('strtolower', $aidstreamOrganizationIdentifierArray);
+            $iatiOrganizations = $this->organizationService->getOrganizationByPublisherIds($aidstreamOrganizationIdentifierArray);
+
+            foreach ($iatiOrganizations as $iatiOrganization) {
+                $recipientCountryBudget = $iatiOrganization->recipient_country_budget;
+
+                if(empty($recipientCountryBudget['status'])){
+                    $recipientCountryBudget['status'] = '1';
+                    $iatiOrganization->timestamps     = false;
+                    $iatiOrganization->updateQuietly(
+                        ['recipient_country_budget'=>$recipientCountryBudget],
+                        ['touch'                   =>false]
+                    );
+                }
+            }
+
+            $this->databaseManager->commit();
+
+            $this->info('Completed updating organization.');
+        } catch(\Exception $e) {
+            $this->databaseManager->rollBack();
+            $this->logInfo($e->getMessage());
+            logger()->error($e);
+        }
+    }
+
+    /**
+     * Ask input from user and return value.
+     *
+     * @param $question
+     * @param $field
+     * @param $rules
+     *
+     * @return string
+     */
+    protected function askValid($question, $field, $rules): string
+    {
+        $value = $this->ask($question);
+        $message = $this->validateInput($rules, $field, $value);
+
+        if ($message) {
+            $this->error($message);
+
+            return $this->askValid($question, $field, $rules);
+        }
+
+        return $value;
+    }
+
+    /**
+     * Validates input given by user.
+     *
+     * @param $rules
+     * @param $fieldName
+     * @param $value
+     *
+     * @return string|null
+     */
+    protected function validateInput($rules, $fieldName, $value): ?string
+    {
+        $validator = Validator::make([
+            $fieldName => $value,
+        ], [
+            $fieldName => $rules,
+        ]);
+
+        return $validator->fails()
+            ? $validator->errors()->first($fieldName)
+            : null;
+    }
+
+    /**
+     * Returns array of [organizationId => organizationIdentifier].
+     *
+     * @param $aidstreamOrganizationIds
+     *
+     * @return array
+     */
+    private function getAidstreamOrganizationIdentifier($aidstreamOrganizationIds): array
```

Comment:
may need to take organization identifier from organization table if settings registry info does not exist